<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>978</width>
    <height>653</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowIcon">
   <iconset theme="thepage" resource="resources.qrc">
    <normaloff>:/icons/thepage.svg</normaloff>:/icons/thepage.svg</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QWidget" name="topWidget" native="true">
      <property name="autoFillBackground">
       <bool>true</bool>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <layout class="QHBoxLayout" name="leftCsdLayout">
         <property name="spacing">
          <number>0</number>
         </property>
        </layout>
       </item>
       <item>
        <widget class="QToolButton" name="menuButton">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset theme="thepage" resource="resources.qrc">
           <normaloff>:/icons/thepage.svg</normaloff>:/icons/thepage.svg</iconset>
         </property>
         <property name="popupMode">
          <enum>QToolButton::InstantPopup</enum>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="tabButtonLayout">
         <property name="spacing">
          <number>0</number>
         </property>
        </layout>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QHBoxLayout" name="jobButtonLayout"/>
       </item>
       <item>
        <layout class="QHBoxLayout" name="rightCsdLayout">
         <property name="spacing">
          <number>0</number>
         </property>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="tStackedWidget" name="stackedWidget"/>
    </item>
   </layout>
  </widget>
  <action name="actionExit">
   <property name="icon">
    <iconset theme="application-exit">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionNew_Tab">
   <property name="icon">
    <iconset theme="tab-new">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>New Tab</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+T</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="icon">
    <iconset theme="document-open"/>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>tStackedWidget</class>
   <extends>QStackedWidget</extends>
   <header location="global">tstackedwidget.h</header>
   <container>1</container>
   <slots>
    <signal>switchingFrame(int)</signal>
   </slots>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
